<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool Tester - AIStudio Quality Assurance</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lbl_title">
      <property name="text">
       <string>üß™ Tool Tester - Automated Testing for AIStudio Tools</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="styleSheet">
       <string>font-size: 18pt; font-weight: bold; padding: 10px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grp_tools">
      <property name="title">
       <string>Registered Tools</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="lst_tools">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lbl_tool_count">
           <property name="text">
            <string>0 tools registered</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_refresh">
           <property name="text">
            <string>üîÑ Refresh</string>
           </property>
           <property name="minimumWidth">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grp_actions">
      <property name="title">
       <string>Test Actions</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_test_selected">
         <property name="text">
          <string>‚ñ∂Ô∏è Test Selected</string>
         </property>
         <property name="minimumHeight">
          <number>40</number>
         </property>
         <property name="styleSheet">
          <string>font-size: 12pt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_test_all">
         <property name="text">
          <string>‚ñ∂Ô∏è Test All Tools</string>
         </property>
         <property name="minimumHeight">
          <number>40</number>
         </property>
         <property name="styleSheet">
          <string>font-size: 12pt; font-weight: bold;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_stop">
         <property name="text">
          <string>‚èπÔ∏è Stop</string>
         </property>
         <property name="minimumHeight">
          <number>40</number>
         </property>
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grp_progress">
      <property name="title">
       <string>Test Progress</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QProgressBar" name="progress_bar">
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_status">
         <property name="text">
          <string>Ready to test</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="grp_results">
      <property name="title">
       <string>Test Results</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lbl_passed">
           <property name="text">
            <string>‚úÖ Passed: 0</string>
           </property>
           <property name="styleSheet">
            <string>color: green; font-weight: bold;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_failed">
           <property name="text">
            <string>‚ùå Failed: 0</string>
           </property>
           <property name="styleSheet">
            <string>color: red; font-weight: bold;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_skipped">
           <property name="text">
            <string>‚äò Skipped: 0</string>
           </property>
           <property name="styleSheet">
            <string>color: gray; font-weight: bold;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txt_log">
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Test results will appear here...</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="btn_clear_log">
           <property name="text">
            <string>üßπ Clear Log</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_export">
           <property name="text">
            <string>üíæ Export Results</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_help">
           <property name="text">
            <string>üìñ Help</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
